Lombiq.Tests.UI/Services/SmtpService.cs

```diff
@@ -72,10 +75,14 @@ public async Task<SmtpServiceRunningContext> StartAsync()
         var manifest = JObject.Parse(await File.ReadAllTextAsync(dotnetToolsConfigFilePath, token));
 
         var smtp4devConfig = (manifest["tools"] as JObject)?["rnwood.smtp4dev"];
+        // We don't need to access smtp4devConfig otherwise, this is just a null check, so using a coalesce expression
+        // would make the code less readable.
+#pragma warning disable IDE0270 // Use coalesce expression
         if (smtp4devConfig == null)
         {
             throw new InvalidOperationException("There was no smtp4dev configuration in the .NET CLI local tool manifest file.");
         }
+#pragma warning restore IDE0270 // Use coalesce expression
```

Comment:
Thinking about this again, it seems to me that a simple comment would make this code simpler:

```suggestion
        // Verify that an smtp4dev configuration is in place.
        if ((manifest["tools"] as JObject)?["rnwood.smtp4dev"] == null)
        {
            throw new InvalidOperationException("There was no smtp4dev configuration in the .NET CLI local tool manifest file.");
        }
```